utils/ESR_Peers_List.cpp

```diff
@@ -0,0 +1,317 @@
+// Copyright (C) 2018 Bluzelle
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License, version 3,
+// as published by the Free Software Foundation.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+// The following curl will get a list of peers for a given swarm id:
+// curl --data '{"jsonrpc":"2.0","method":"eth_call","params":[{"to":"0x3e4BDd757322eF358664090c69b6927BdB6cf2De","data": "0x46e76d8b0000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000000d426c757a656c6c65537761726d00000000000000000000000000000000000000" },"latest"],"id":1}' https://ropsten.infura.io/uvek7IebbbHoP8Bb9NkV
+
+
+// Given a swarm id and a node address, the following curl will get the node info
+// curl --data '{"jsonrpc":"2.0","method":"eth_call","params":[{"to":"0x3e4BDd757322eF358664090c69b6927BdB6cf2De","data": "0xcc8575cb00000000000000000000000000000000000000000000000000000000000000400000000000000000000000000000000000000000000000000000000000000080000000000000000000000000000000000000000000000000000000000000000d426c757a656c6c65537761726d00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000c3230392e35302e36312e38370000000000000000000000000000000000000000" },"latest"],"id":1}' https://ropsten.infura.io/uvek7IebbbHoP8Bb9NkV
```

Comment:
wrap please!

Reply:
Done.. Would it be better if I just removed the curl command examples?

Reply:
Yes, removing the examples. I'll put them in the associated JIRA issue.